test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua

```diff
@@ -135,7 +135,7 @@ function Test:Precondition_PTU_revoke_app_group()
             {status = "UPDATING"}, {status = "UP_TO_DATE"}):Times(2)
           :Do(function(_,data)
               if(data.params.status == "UP_TO_DATE") then
-                EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = HMIAppID, isAppPermissionsRevoked = true, appRevokedPermissions = {"DataConsent"}})
+                EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = HMIAppID, isAppPermissionsRevoked = true, appRevokedPermissions = { {name = "DrivingCharacteristics"} } })
```

Comment:
@VVeremjova as requirement is JIRA ID, I can't list it here:
The description: [Policies] SDL.ActivateApp from HMI and 'isAppPermissionsRevoked' parameter in the response